core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -5,92 +5,9 @@
-              <div class="col-lg-10 col-md-10 col-sm-10">
-                <select id="explorationLanguageCode" class="form-control protractor-test-exploration-language-select" ng-model="explorationLanguageCodeService.displayed" ng-change="saveExplorationLanguageCode()" ng-options="lc.code as lc.description for lc in explorationLanguageCodeService.getAllLanguageCodes()">
-                </select>
-                <span class="help-block" style="font-size: smaller">
-                  <em>
-                    Don't see the language you want? <a href="https://github.com/oppia/oppia/issues/new?title=Please%20add%20a%20new%20language%20choice%20to%20the%20exploration%20settings%20tab&body=Please%20add%20the%20language%20choice%20%7B%7BYOUR_LANGUAGE_HERE%7D%7D%20to%20the%20exploration%20settings%20tab.%0A%0AHere%20is%20a%20link%20to%20an%20exploration%20that%20uses%20it:%20%7B%7BINSERT_LINK_HERE%7D%7D" target="_blank">Tell us.</a>
-                  </em>
-                </span>
-              </div>
-            </div>
-            <div class="form-group">
-              <label for="explorationInitStateName" class="col-lg-2 col-md-2 col-sm-2">Name of first card</label>
-              <div class="col-lg-10 col-md-10 col-sm-10">
-                <select id="explorationInitStateName" class="form-control protractor-test-initial-state-select" ng-model="explorationInitStateNameService.displayed" ng-change="saveExplorationInitStateName()" ng-options="name as name for name in stateNames track by name">
-                </select>
-                <span class="help-block" style="font-size: smaller">
-                  <em>This is the first card the learner sees when playing your exploration.</em>
-                </span>
-              </div>
-            </div>
-            <div class="form-group">
-              <label for="explorationTags" class="col-lg-2 col-md-2 col-sm-2">Tags</label>
-              <div class="col-lg-10 col-md-10 col-sm-10">
-                <div ng-if="hasPageLoaded">
-                  <select2-dropdown
-                    item="$parent.explorationTagsService.displayed"
-                    tag-mode="true" allow-multiple-choices="true"
-                    invalid-search-term-message="Add a new tag (using lowercase letters and spaces)..."
-                    new-choice-regex="<[::TAG_REGEX]>" width="100%"
-                    placeholder="Skills, concepts, topics, ..."
-                    on-selection-change="saveExplorationTags()">
-                  </select2-dropdown>
-                </div>
-                <span class="help-block" style="font-size: smaller">
-                  <em>Tags help learners discover your exploration when searching.</em>
-                </span>
-              </div>
-            </div>
+            <exploration-metadata-form 
+            ng-init="formStyle={'display':'table-cell', 'width':'16.66666667%'}; pStyle={'display': 'table', 'width':'100%', 'margin-bottom':'2%'};">
```

Comment:
Hm, this is an odd way of passing in arguments. Can we instead do:

```<exploration-metadata-form form-style="..." p-style="..."```

and read them in the JS controller for that directive using '@', and then pass them on explicitly? I think what you're doing here is a bit too indirect, and it'll be hard to maintain since we don't have an explicit interface for what gets passed into the directive.